Source/cmake/OptionsJSCOnly.cmake

```diff
@@ -55,10 +57,14 @@ endif ()
 
 # FIXME: JSCOnly on WIN32 seems to only work with fully static build
 # https://bugs.webkit.org/show_bug.cgi?id=172862
-if (NOT ENABLE_STATIC_JSC AND NOT WIN32)
-    set(JavaScriptCore_LIBRARY_TYPE SHARED)
-    set(bmalloc_LIBRARY_TYPE OBJECT)
-    set(WTF_LIBRARY_TYPE OBJECT)
+if (NOT ENABLE_STATIC_JSC)
+      set(JavaScriptCore_LIBRARY_TYPE SHARED)
+    if (NOT WIN32)
+      set(bmalloc_LIBRARY_TYPE OBJECT)
+      set(WTF_LIBRARY_TYPE OBJECT)
+    else ()
+      set(WTF_LIBRARY_TYPE SHARED)
+    endif ()
```

Comment:
Can you just remove the `AND NOT WIN32` and the above FIXME? If it works that would be my preference.

Reply:
It is not fixed. We are reusing what `WinCairo` port is doing.
So we can not make it uniform across platforms for now.